tests/rkt_tests.go

```diff
@@ -133,19 +135,28 @@ func (ctx *rktRunCtx) dir(idx int) string {
 }
 
 func (ctx *rktRunCtx) reset() {
+	ctx.cleanupChildren()
 	ctx.runGC()
 	for _, d := range ctx.directories {
 		d.reset()
 	}
 }
 
+func (ctx *rktRunCtx) cleanupChildren() {
+	for _, child := range ctx.children {
+		log.Printf("Shutting down %q", child.Cmd.Path)
```

Comment:
I didn't see it used anywhere else in that file, but that's not a real reason so I'll make it work.